targets/standalone/index.js

```diff
@@ -3,30 +3,18 @@
 /**
  * Internal dependencies
  */
-import { initHappychat, eventAPI } from 'src';
+import api from 'src/api';
 import config from 'targets/standalone/config';
 
 const wpcomOAuth = require( 'wpcom-oauth-cors' )( config( 'oauth_client_id' ) );
 
-const accessToken = () =>
-	new Promise( ( resolve, reject ) => {
+// expose the access token promise
+window.HappychatAccessTokenPromise = () =>
+	new Promise( ( resolve ) => {
```

Comment:
How come we are not using `async/await`?

Reply:
That would be a nice to have but considering our latest change in priorities, I'm trying to reuse most of the code and only refactor parts that are needed so that we can integrate with Calypso.